sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubClient.cs

```diff
@@ -208,6 +208,37 @@ public override async Task<EventHubProperties> GetPropertiesAsync(CancellationTo
             );
         }
 
+        /// <summary>
+        ///   Creates an event sender responsible for transmitting <see cref="EventData" /> to the
+        ///   Event Hub, grouped together in batches.  Depending on the <paramref name="senderOptions"/>
+        ///   specified, the sender may be created to allow event data to be automatically routed to an available
+        ///   partition or specific to a partition.
+        /// </summary>
+        ///
+        /// <param name="senderOptions">The set of options to apply when creating the sender.</param>
+        ///
+        /// <returns>An event sender configured in the requested manner.</returns>
+        ///
+        public override EventSender CreateSender(EventSenderOptions senderOptions)
+        {
+           (TimeSpan minBackoff, TimeSpan maxBackoff, int maxRetries) = ((ExponentialRetry)senderOptions.Retry).GetProperties();
+
+           TrackOne.EventDataSender createSender()
```

Comment:
suggestion: rename this to `createSenderFactory`?

Reply:
Will do;  that makes sense.

Reply:
Done, and fixed casing as well.